Últimos acertos na app RaulLino

GoncaloAlpalhao opened the pull request:
# Parabéns a todo o grupo pela nossa app ter sido a escolhida para integrar o projeto. Se não fosse o trabalho de todos isto não teria acontecido 💪 
Estava aborrecido e ao ver o email do professor a pedir alterações ao trabalho decidi usar isso como entretenimento (é estranho alguém entreter-se com kotlin eu sei mas pronto, cada um com as suas manias 😂), tentei ao máximo seguir as alterações que o professor indicou tentando não criar quaisquer outros bugs.

Peço a todos que revejam se a app está de acordo com os novos acertos e que não tenha bugs ou tenha comprometido alguma funcionalidade anteriormente existente. Tentei com que os textos em inglês também se adaptassem a estas novas alterações.

> Nota: O link na homepage é clicável e redireciona o utilizado para a página indicada.

GoncaloAlpalhao committed:
Home Redesenhada

Foram retirados todos os elementos antigos e foram colocados:
Um banner como topo do fragmento
O texto fornecido pelo professor
Um link clicável que redireciona para a página indicada
Os logótipos e texto respetivos.
Foi garantido o funcionamento do mesmo na orientação horizontal e multilingua

GoncaloAlpalhao committed:
Alterações Biografia

Refactor às strings para melhor se adaptarem aos requisitos

GoncaloAlpalhao committed:
Alteração Detalhe Edificio

Foi retirada a fonte do projeto no título do detalhe dos edificios devido a problemas com caracteres não ingleses e números.
Foi ajustado o tamanho da letra para melhor se adequear
Um ligeiro ajuste à nota biográfica por estética

hgoncal2 reviewed (approved):
Eu testei e parece-me tudo bem! Só reparei num pormenor,que é o de não ser imediatamente óbvio que existe um texto por baixo dos logótipos,pelo menos no emulador.
![image](https://github.com/iptomar/RaulLinoA/assets/101368171/1dd4c48e-89ab-489f-86dc-02eee72cb655)
Mas vou aprovar à mesma porque está de acordo com as especificações do professor.

E obrigado pelo teu trabalho durante todo este projeto!

GoncaloAlpalhao commented:
Thanks pela observação! Eu tenho esse layout com altura estática para se adaptar em modo horizontal (ScrollView) mas posso diminuir a margem entre o URL e os logótipos para se adaptar a mais dispositivos.

GoncaloAlpalhao committed:
Ligeiro acerto na home screen

GoncaloAlpalhao committed:
Correção das Traduções

Foram alteradas as traduções em inglês
Foi também corrigido o marker da rua luís de camões que estava incorreto no json inglês
